packages/lib-classifier/dev/components/App/App.js

```diff
@@ -173,20 +173,27 @@ class App extends React.Component {
     const mergedThemes = _.merge({}, baseTheme, zooTheme, { dark: this.state.dark })
     const key = this.state.cachePanoptesData ? 'cachedClassifier' : 'classifier'
 
+    const availableLocales = project?.available_languages.map(current => {
+      let newLocale = current
+      if (current === 'zh-cn') newLocale = 'zh-CN'
+      if (current === 'zh-tw') newLocale = 'zh-TW'
+      return newLocale
```

Comment:
Same comment here about generalising this function to any two-part language code.